packages/ui/src/index.ts

```diff
@@ -1,7 +1,25 @@
+export * from './Alert'
+export * from './Badge'
+export * from './Button'
+export * from './Checkbox'
+export * from './CheckboxFormik'
 export * from './Error'
 export * from './Help'
 export * from './Icon'
+export * from './IconButton'
+export * from './Label'
```

Comment:
I would suggest leaving `Label` private

Reply:
@aigoncharov let's say we will implement project specific components, this can be handy. 

e.g. maybe we will need this for card input (we will use Stripe's one) & connect this label, I think we could use this.